libs/components/packages/src/schematics/utility/json-file.ts

```diff
@@ -101,7 +101,7 @@ export class JsonFile {
 
     this.content = applyEdits(this.content, edits);
     this.host.overwrite(this.path, this.content);
-    this._jsonAst = undefined;
+    this.#_jsonAst = undefined;
```

Comment:
I know this isn't your fault, but we shouldn't be setting the property directly and should instead use the setter, `this.#jsonAst = undefined`.

Reply:
Looking at this some more I didn't like how the getter was working here. It wasn't really a getter and was going a lot of logic. Since it was a private getter I just changed it to be a function and modified the variable to not use the underscore. Let me know if my changes don't make sense.